statsmodels/examples/ex_kernel_test_functional_li_wang.py

```diff
@@ -99,7 +97,7 @@
         #res_ols = mod_ols.fit()
         #'cv_ls'[1000, 0.5]
         bw_lw = [1./np.sqrt(12.) * nobs**(-0.2)]*2  #(-1. / 5.)
-        tst = smke.TestFForm(endog, exog[:,:1], bw=bw_lw, var_type='c',
+        tst = smke.TestFForm(endog, exog, bw=bw_lw, var_type='c'*len(bw_lw),
```

Comment:
Note to self: double check this change; it is sufficient to make the code not-raise, but not necessarily correct.